CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1576,18 +1576,21 @@ private void PresentationSourceChangedHandler(object sender, SourceChangedEventA
                     }
 
                     //Ignore this for custom bitmap factories
-                    if (RenderHandler != null && RenderHandler.GetType() == typeof(WritableBitmapRenderHandler) || RenderHandler.GetType() == typeof(InteropBitmapRenderHandler))
+                    if (RenderHandler != null)
```

Comment:
This is now redundant and can be removed

Reply:
removed